partcad/src/partcad/globals.py

```
@@ -35,7 +35,7 @@
 factory.register("provider", "store", ProviderFactoryStore)
 
 
-def init(config_path=None, search_root=True) -> Context:
+def init(config_path=None, search_root=True, transaction=None) -> Context:
```

Ruff (0.8.2)

Missing type annotation for function argument `config_path`

(ANN001)